scripts/tests/auth/test-self-hosted.sh

```diff
@@ -110,10 +110,10 @@ rm -rf odd-project
 # test that earthly has access to it
 "$earthly" config git "{myserver: {pattern: 'myserver/([^/]+)', substitute: 'ssh://root@$ip:2222/root/my/really/weird/path/\$1.git', auth: ssh}}"
 
-echo === Test remote build under repo root ===
+echo "=== Test remote build under repo root ==="
```

Comment:
This is most of the lint issues the shellcheck stage is finding on the M1. Wonder if its due to the one in Alpine's repo being old? Maybe migrate the other platforms over to using a manual download installation?